doc/prometheus_vm_microstates_collector.md

```diff
@@ -0,0 +1,37 @@
+
+
+# Module prometheus_vm_microstates_collector #
+* [Description](#description)
+
+Collects statistics about the microstate accounting of Erlang VM threads using
+[msacc](http://erlang.org/doc/man/msacc.md).
+
+__Behaviours:__ [`prometheus_collector`](prometheus_collector.md).
+
+<a name="description"></a>
+
+## Description ##
+
+Refer to
+[
+erlang:statistics(microstate_accounting)
+](http://erlang.org/doc/man/erlang.md#statistics_microstate_accounting) for more information about thread types and microstates.
```

Comment:
```suggestion
](http://erlang.org/doc/man/erlang.html#statistics_microstate_accounting) for more information about thread types and microstates.
```

I don't know where you got these `.md` links from but they return 404 for me but `.html` works :)